EAS energy model source: https://github.com/chips-project/kernel_asus_sdm660/commit/572b3c9494e8e38f09873646e031e16621dcd3fc
By Danny Lin <danny@kdrag0n.dev>
--- 
+++ 
@@ -17067,28 +17067,28 @@
 		compatible = "sched-energy";
 
 		core-cost0 {
-			busy-cost-data = <0x9ab00 0x29 0xdc500 0x46 0x10fe00 0x53 0x156300 0x92 0x177000 0x9e 0x189c00 0xb7>;
+			busy-cost-data = <0x9ab00 0x04 0xdc500 0x09 0x10fe00 0x0e 0x156300 0x17 0x177000 0x20 0x189c00 0x28>;
 			idle-cost-data = <0x14 0x10 0x0c 0x08>;
 			linux,phandle = <0x05>;
 			phandle = <0x05>;
 		};
 
 		core-cost1 {
-			busy-cost-data = <0x10fe00 0x133 0x156300 0x1e5 0x1aa900 0x359 0x1b8a00 0x373>;
+			busy-cost-data = <0x10fe00 0x6f 0x156300 0x8c 0x1aa900 0xae 0x1b8a00 0xb4>;
 			idle-cost-data = <0x64 0x50 0x3c 0x28>;
 			linux,phandle = <0x0e>;
 			phandle = <0x0e>;
 		};
 
 		cluster-cost0 {
-			busy-cost-data = <0x9ab00 0x04 0xdc500 0x05 0x10fe00 0x07 0x156300 0x09 0x177000 0x09 0x189c00 0x0a>;
+			busy-cost-data = <0x9ab00 0x03 0xdc500 0x08 0x10fe00 0x0d 0x156300 0x17 0x177000 0x20 0x189c00 0x27>;
 			idle-cost-data = <0x04 0x03 0x02 0x01>;
 			linux,phandle = <0x06>;
 			phandle = <0x06>;
 		};
 
 		cluster-cost1 {
-			busy-cost-data = <0x10fe00 0x0e 0x156300 0x11 0x1aa900 0x19 0x1b8a00 0x1f>;
+			busy-cost-data = <0x10fe00 0x22 0x156300 0x3f 0x1aa900 0x6f 0x1b8a00 0xb2>;
 			idle-cost-data = <0x04 0x03 0x02 0x01>;
 			linux,phandle = <0x0f>;
 			phandle = <0x0f>;
